### PR TITLE
Method maple gmoranzoni

### PR DESCRIPTION
**Pull request by gmoranzoni**
updated yml file

### CODE REVIEW

**Comment by niklasmueboe** [comment]
Why are these changes necessary? What do they fix?

**Comment by gmoranzoni** [comment]
The Seurat version was wrong (5.0.1 instead of 4.4.0), and therefore the installation of the maple environment would fail. Now it should be fixed.

File: method/maple/maple_env.sh
Comment (niklasmueboe) [inline method/maple/maple_env.sh:1]: Is the spruce package necessary?

File: method/maple/maple.r [hunk 1]
```diff
@@ -174,7 +174,7 @@ seurat_obj@images$image =  new(
   Class = 'SlideSeq',
   assay = "Spatial",
   key = "coordinates_",
-  coordinates = as.data.frame(spatialCoords(spe))
+  coordinates = as.data.frame(colData(spe))
```
Comment (niklasmueboe) [inline method/maple/maple.r:177]: Cordinates are stored in the spatialCoords, in the colData you will only find array rows/cols for ST/Visium datasets. Is this intended?

Either way you should probably explicitly extract the relvant columns from colData as there could be more in the future

File: method/maple/maple_env.sh
Comment (gmoranzoni) [reply method/maple/maple_env.sh:1]: Yes, it is a maple dependency.

**Review by niklasmueboe — APPROVED** [review]

**Pull request merged**